src/chartCore/getRange.js

```diff
@@ -11,7 +11,7 @@ export default function getRange(props, key) {
     case 'y':
       return [getMaxY(plotRect), plotRect.y]
     case 'radius':
-      return type === 'ordinal' ? [2, 4, 8, 12, 16, 20] : [2, 20]
+      return type === 'ordinal' ? [2, 4, 8, 12, 16, 20] : [1, 20]
```

Comment:
Can you explain this change, for posterity?

Reply:
I'm not necessarily sure this is the right change to make. I think the question that should be answered is what do we expect to see on tooltip hover for a dimension value that is not `x` or `y`. Do you expect to see the value given as input or would you expect to see the scaled value as it is displayed on the chart?

For example, passing a `fill={10}` value would you expect to see `value 10.000` or `value rgb(r, g, b)`?

Reply:
Without having thought about it too carefully, I think you should always see the value given as input. The displayed value is just a visual representation, but the tooltip should show the actual datum.

Reply:
Right, I think that makes sense (and is also the current behavior). Given that, I think it's best to close this (and the issue) since the current code is doing what we expect. We can make an additional PR that adds the `tooltipShowKeys` to the example charts where need be.

Reply:
👌 gotcha, thanks for the explanation and for looking into this!